minimal/docker-compose.yml

```diff
@@ -53,4 +53,13 @@ services:
     volumes_from:
       - {{ project_name }}_web
     ports:
-      - "0.0.0.0:80:80"
\ No newline at end of file
+      - "0.0.0.0:80:80"
+
+  {{ project_name }}_certbot:
+    restart: always
+    build: ./compose/certbot/production
```

Comment:
@pslobo Having things properly separated (even if they are reduntant) seems like a good idea to me, because users can easily customize for a certaing enviroment. And you also enforce the need for different credentials and settings for each environment